packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -226,4 +226,85 @@ export const TIME_SERIES_METRIC_FIELD_NAME = 'Metric';
 export interface DataFrameFieldIndex {
   frameIndex: number;
   fieldIndex: number;
+  seriesIndex?: number;
+  displayName?: string;
+}
+
+/**
+ * @internal
+ */
+export interface FieldMap {
```

Comment:
not sure where this should live -- but for sure not with `dataFrame` -- if it can be internal to viz stuff that would be best,  this is really a way for a few of our visualizations to share semantic intent, NOT a new data format/structure